tests/conftest.py

```diff
@@ -888,6 +888,10 @@ def fixture_default_simplex_config():
 @pytest.fixture(name="onnx_str")
 def fixture_default_onnx_str() -> bytes:
     """The default ONNX model string to be used if not specified differently."""
+    # Since the onnx import is done later in the tests bur this fixture uses
```

Comment:
Typo in "but".

However, I also suggest that you rewrite this comment, because it's completely unclear what "the deprecated function" is, when you realize that you will later look at this text without having the PR context. I would reformulate and mention to the import is needed to trigger the monkeypatching applied in the `_optional.onnx` module. And because there will then be an explanation (see my other thread), the reasoning will be clear to the reader